lib/test/t_FunctionalChaosSobolIndices_ishigami.cxx

```diff
@@ -19,6 +19,7 @@
  *
  */
 #include "openturns/OT.hxx"
+#include "openturns/OTIshigamiUseCase.hxx"
```

Comment:
You should drop the OT prefixes in the file names: GSobolUseCase.hxx, IshigamiUseCase.hxx

Reply:
Ok. Fixed.